collect_app/src/test/java/org/odk/collect/android/utilities/TextUtilsTest.java

```diff
@@ -0,0 +1,49 @@
+package org.odk.collect.android.utilities;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+
+@RunWith(RobolectricTestRunner.class)
+public class TextUtilsTest {
+
+    /**
+     * Should return null if provided with null and not throw a NPE.
+     *
+     * This is a silly test but it is here to guarantee this behaviour,
+     * since without it the method causes a crash when processing text for
+     * questions with no plain text label. See opendatakit/opendatakit#1247.
+     */
+    @Test
+    public void textToHtml_BouncesNullInput() {
+        String input = null;
+        CharSequence observed = TextUtils.textToHtml(input);
+        assertNull(observed);
+    }
+
+    /**
+     * Should convert Markdown to HTML.
+     *
+     * In the real app this call would produce the following string, but
+     * because the implementation uses android.Html.fromHtml, we end up with
+     * robolectric's implementation of fromHtml which isn't exactly the same.
+     * See opendatakit/collect#204.
```

Comment:
Let's not let perfect be the enemy of good. @lindsay-stevens remove the HTML test and then I'll be happy to merge.